client/src/main/java/org/jboss/fuse/mvnd/client/Installer.java

```diff
@@ -159,11 +164,15 @@ static void unzip(Path localZip, Path destinationDir) {
     static Path download(URI zipUri) {
         try {
             final Path localZip = Files.createTempFile("", "-mvnd-dist.zip");
-            LOG.debug("Downloading {} to {}", zipUri, localZip);
+            if (LOG.isInfoEnabled()) {
```

Comment:
Same as above

Reply:
Fixed in 832e1ecc150949de5348b0550179e0d04d5df7b0